feat: widget terms and conditions

shoom3301 opened the pull request:
Source: https://docs.google.com/document/d/11MFz0nMYCtv3ztOgXh9lMxfIk2PAhfCYxo3vasMiReI/edit
Context: https://cowservices.slack.com/archives/C05QVFJUX9R/p1710942507985319

I've used https://www.gdoctohtml.com/ to convert google doc to HTML

elena-zh reviewed (commented):
Hey @shoom3301 , great!

Some nitpicks: 
1. The page is hard to find, Maybe add a button like Ordan suggests? 
![image](https://github.com/cowprotocol/cow-fi/assets/70885163/f8eeaab5-0c2b-4afe-8c5b-c730e5379b3d)
2. Proposal from my side is to add a link to these terms page into the footer 
![image](https://github.com/cowprotocol/cow-fi/assets/70885163/685b3d07-e5bd-4f21-8df4-86a74a7585d9)
3. There should be a period in the end 
![period](https://github.com/cowprotocol/cow-fi/assets/70885163/cb9ac3b1-76e2-4fa0-b600-2ddbeb1df737)
4. Looks like space is missing here 
![space](https://github.com/cowprotocol/cow-fi/assets/70885163/126f48be-c8c2-4d7f-b691-41cad553d2c6)
5. It would be great to add a mailto link here
![mailto](https://github.com/cowprotocol/cow-fi/assets/70885163/c44a71b8-df9e-4ef3-b7bc-535f71f8c0f1)

Thanks!

alfetopito commented on pages/index.tsx:
```diff
@@ -28,7 +28,17 @@ export default function HomePage({ metricsData, siteConfigData }: HomeProps) {
+  let volumeUsd = 0
+  let volumeEth = 0
+
+  // Don't fail when couldn't get Subgraph data
```
Shouldn't this be an isolated PR in itself?
Then we can release it independently from the T&C

shoom3301 replied:
https://github.com/cowprotocol/cow-fi/pull/170

shoom3301 commented:
@elena-zh thank you!

1. There is a link in "Important Disclaimer" message. Have Odran mentioned another place we should put the link? Sorry, probably I missed that part
2. Makes sense, added it
3. Fixed
4. Fixed
5. Fixed

<img width="864" alt="image" src="https://github.com/cowprotocol/cow-fi/assets/7122625/e0cab0ff-9592-4cdc-a3f6-b4ad9ddabb1d">

elena-zh commented:
Hey @shoom3301 , 
1. yes, Ordan mentioned about it in https://cowservices.slack.com/archives/C05QVFJUX9R/p1710942507985319
2. case 5 is not fixed :( 
![image](https://github.com/cowprotocol/cow-fi/assets/70885163/bdaea715-b886-447c-aa85-92f0eb211134)
3. And could you start Terms and Conditions words in the footer with capitals? 
![image](https://github.com/cowprotocol/cow-fi/assets/70885163/ad92d9f4-516d-4a4b-8288-fd14cdfc9490)

Thanks!

elena-zh reviewed (approved):
Great, thank you!

elena-zh commented:
Hey @alfetopito , paragraphs order  is mixed up here
![image](https://github.com/cowprotocol/cow-fi/assets/70885163/98a3d706-e4f9-47a7-a334-3b3b0cb8a352)

Could you please fix this?

alfetopito commented:
> Hey @alfetopito , paragraphs order is mixed up here ![image](https://private-user-images.githubusercontent.com/70885163/316545887-98a3d706-e4f9-47a7-a334-3b3b0cb8a352.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTEzNzk1MTQsIm5iZiI6MTcxMTM3OTIxNCwicGF0aCI6Ii83MDg4NTE2My8zMTY1NDU4ODctOThhM2Q3MDYtZTRmOS00N2E3LWEzMzQtM2IzYjBjYjhhMzUyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzI1VDE1MDY1NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTg3ODRlZTUwNGYyZDYwY2VkYTBmNjFkYzBiM2NiOGNmYWRhM2Y3MmYwNjkyM2RlMjE2YmY4YjMwMjYyNjJjZDAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.KH6f0sxu8z3p1kGt66mjLD14dT_9ZnnZOMLYQGz_SzQ)
> 
> Could you please fix this?

Oops, git commit mess up. Fixed

elena-zh commented:
Looks good to me now!
